test/dbapi/conftest.py

```diff
@@ -0,0 +1,95 @@
+import os
+from collections import ChainMap
+from dataclasses import dataclass
+
+import pytest
+
+
+def _option_name(name: str):
+    return f"--exasol-{name}"
+
+
+def _envvar_name(name: str):
+    return f"EXASOL_{name.upper()}"
+
+
+_OPTIONS = ["port", "host", "username", "password", "schema"]
+
+_DEFAULTS = {
+    _envvar_name("port"): 8888,
+    _envvar_name("host"): "localhost",
+    _envvar_name("username"): "SYS",
+    _envvar_name("password"): "exasol",
+    _envvar_name("schema"): "TEST",
+}
+
+
+def pytest_addoption(parser):
```

Comment:
not used method.

Reply:
By putting this function into conftest it will register options which can be passed to pytest.
![image](https://user-images.githubusercontent.com/600911/215691100-ab8d3c69-7730-413c-87ac-f53a26b29a6a.png)